iree/compiler/Translation/IREEVM.cpp

```diff
@@ -103,9 +103,11 @@ static LogicalResult translateFromMLIRToVMBytecodeModuleWithFlags(
                                              bytecodeTargetOptions, output);
 }
 
-static TranslateFromMLIRRegistration toVMBytecodeModuleWithFlags(
-    "iree-mlir-to-vm-bytecode-module",
-    translateFromMLIRToVMBytecodeModuleWithFlags);
+void registerIREEVMTranslation() {
```

Comment:
I'm wondering about declarations and definitions in these registrations. Should this be declared in [IREEVM.h](https://github.com/google/iree/blob/master/iree/compiler/Translation/IREEVM.h), as `registerIREEVMTransformPassPipeline()` is? Then `init_translations.h` might include `IREEVM.h` instead of forward declaring? That seems less prone to link-time errors.

Maybe I'm still sleepy and getting my C++ confused...

Reply:
No, you're right. I think I implemented `registerIREEVMTransformPassPipeline()` in a later commit (which was merged before this one) and decided to go with "your suggestion". So I will refactor this one :)